fvp.mk

```diff
@@ -141,6 +146,18 @@ else
 		      EVENT_LOG_LEVEL=20
 endif
 
+TF_A_FLAGS_BL32_OPTEE  = BL32=$(OPTEE_OS_HEADER_V2_BIN)
+TF_A_FLAGS_BL32_OPTEE += BL32_EXTRA1=$(OPTEE_OS_PAGER_V2_BIN)
+TF_A_FLAGS_BL32_OPTEE += BL32_EXTRA2=$(OPTEE_OS_PAGEABLE_V2_BIN)
+TF_A_FLAGS_BL32_OPTEE += ARM_TSP_RAM_LOCATION=tdram
+
+TF_A_FLAGS_SPMC_AT_EL_n  = $(TF_A_FLAGS_BL32_OPTEE) SPD=opteed
+TF_A_FLAGS_SPMC_AT_EL_1  = BL32=$(OPTEE_OS_PAGER_V2_BIN) SPD=spmd
```

Comment:
Why is this different from `qemu_v8.mk`?

Reply:
For some reason passing the header doesn't work on FVP.

Reply:
Thanks. I asked Balint to take a look. fvp-psa-sp.mk should be fine but can get a cleanup as well with these changes. If these changes don't break anything there(a quick look suggested so but would like to hear from Balint), then the cleanup there can be a separate change.

Reply:
Currently this is breaking `fvp-psa-sp.mk`. Because `SPMC_AT_EL` is not set there, it'll default to `n` so the `FF-A`-less TF-A flags are appended to the flags defined in `fvp-psa-sp.mk`